GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -58,9 +69,10 @@ public static void GenerateLoadingScreenScene(string loadingScreenImagePath, str
         //TODO: get rid of error message associated with using the LoadingScreenScript reference
         private static void AddLoadingScreenScript(GameObject loadingScreenGameObject)
         {
-            loadingScreenGameObject.AddComponent<LoadingScreenScript>();
+//            loadingScreenGameObject.AddComponent<LoadingScreenScript>();
```

Comment:
Let's not submit with code commented out. Please uncomment or delete and replace with a TODO.